content/docs/welcome-guide/create/creating-projects-using-project-manager.md

```diff
@@ -79,8 +83,8 @@ You are now ready to build the project from Project Manager.
 
```

Comment:
```suggestion
1. Under **My Projects**, choose **Build Project** for your new project.
```
For the line above this, add a point of reference (for example, "Under **My Projects**") and drop "the" and "button".